layers/+tools/dap/README.org

```diff
@@ -0,0 +1,102 @@
+#+TITLE: DAP layer
+
+* Table of Contents                      :TOC_4_gh:noexport:
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#configuration][Configuration]]
+  - [[#derived-layers][Derived layers]]
+    - [[#declared-prefixes][Declared prefixes]]
+    - [[#default-keybindings][Default keybindings]]
+- [[#references][References]]
+
+* Description
+  Experimental integrated visual debugger using [[https://code.visualstudio.com/docs/extensionAPI/api-debugging][Debug Adapter Protocol]].
+
+  [[https://code.visualstudio.com/docs/extensionAPI/api-debugging][Debug Adapter Protocol]] is a wire protocol for communication between client and
+  Debug Server. It similar to the [[https://github.com/Microsoft/language-server-protocol][LSP]] but providers integration with debug
+  server.
+
+** Features:
+   Fully featured IDE-like debugger providing:
+   - Launch/Attach
+   - Breakpoints
+   - Exceptions
+   - Pause & Continue
+   - Step In/Out/Over
+   - Callstacks
+   - Threads
+   - Multiple simultaneous debug sessions
+   - Evaluating statements
+   - Debug/Run configurations
+   - Debug REPL
+* Configuration
+** Derived layers
+   Derived layer should provide information how to configure the language
+   specific debug adapter and the language specific bindings(e. g. debug
+   application, debug test, etc). The package declares function
+   ~spacemacs/dap-bind-keys-for-mode~ which should be used by the derived layers
+   to configure the shortcuts.
+*** Declared prefixes
+    The following prefixes have been declared:
+
+    | prefix  | name              |
+    |---------+-------------------|
+    | ~m d~   | debug             |
```

Comment:
Please add `SPC` for each of these keybindings in the first column of the table. We have this everywhere else and it makes searching easier.